app/model/table/ComputerTable.scala

```diff
@@ -16,7 +16,7 @@ class ComputerTable(tag: Tag) extends Table[Computer](tag, "computer") {
   def name = column[Option[String]]("name")
 
   // Room foreign key
-  def room: ForeignKeyQuery[RoomTable, Room] = foreignKey("room_id", roomId, TableQuery[RoomTable])(_.id, onUpdate = ForeignKeyAction.Restrict, onDelete = ForeignKeyAction.Cascade)
+  def room: ForeignKeyQuery[RoomTable, Room] = foreignKey("room_id", roomId, TableQuery[RoomTable])(_.id?, onUpdate = ForeignKeyAction.Restrict, onDelete = ForeignKeyAction.Cascade)
```

Comment:
It is a lifting method offered by Slick.

You can see the API definition [here](http://slick.lightbend.com/doc/3.1.1/api/index.html#slick.lifted.BaseColumnExtensionMethods@?:slick.lifted.Rep[Option[P1]])

In this case the `Rep[Long]` instance is lifted to `Rep[Option[Long]]`.

Hope it helps! :smile: